tools/custom-organization-policy-library/build/custom-constraints/gke/gkeDisableLegacyMetadataEndpoints.yaml

```diff
@@ -3,14 +3,13 @@
 
 #@ if constraint.to_generate():
 name: #@ constraint.constraint_name()
-resource_types: 
-- container.googleapis.com/Cluster
-- container.googleapis.com/NodePool 
-condition: 1 == 0
-action_type: DENY
-method_types: 
+resourceTypes:
+- container.googleapis.com/NodePool
+methodTypes:
 - CREATE
-- UPDATE 
+- UPDATE
+condition: ('disable-legacy-endpoints' in resource.config.metadata && resource.config.metadata['disable-legacy-endpoints']== 'false')
```

Comment:
U can remove the first and last bracket
Also please put a space here `y-endpoints'] == 'false')`

Reply:
removing the bracket causes an error. ERROR: (gcloud.org-policies.set-custom-constraint) Unable to parse file [gkeDisableLegacyMetadataEndpointsAndrewTest.yml]: Expecting value: line 1 column 1 (char 0).

Reply:
Yes I tested removing the bracket () and it will cause an error. Adding the spacing did not cause any errors.